### DISCUSSION TITLE
trouble transferring a domain

### DISCUSSION THREAD

**Discussion by MevlutArslan** [vercel/community#6070] [Account Issue]
### Question

Hi

I am having trouble transferring a domain I own "mevlutarslan.com", I keep getting:
"The domain `mevlutarslan.com` already exists".

Can you clarify what this means?
I tried to buy the domain via Vercel's domains but it didn't let me then I bought it via another provider.

Have a nice day,
Mevlut Arslan

**Comment by amsimon** [comment]
I'm having the same issue. Any update on this?
